create_vault.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
The file should be named `create_vaults.yml` because we will generate multiple vaults (one per app) for a project (customer/env_type).